### PR TITLE
feat: Add CITATION.cff file

### PR DESCRIPTION
**Pull request by matthewfeickert**
Seems appropriate that for a talk on software citation we have a `CITATION.cff` file for the talk as well. :) 

```
* Add CITATION.cff for CHEP 2023 talk
   - c.f. https://indico.jlab.org/event/459/contributions/11688/
```

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+message: "Please cite the following works when using this software."
+authors:
+- family-names: "Feickert"
+  given-names: "Matthew"
+  orcid: "https://orcid.org/0000-0003-4124-7862"
+  affiliation: "University of Wisconsin-Madison"
+- family-names: "Katz"
+  given-names: "Daniel"
+  orcid: "https://orcid.org/0000-0001-5934-7525"
+  affiliation: "National Center for Supercomputing Applications"
```
Comment (matthewfeickert) [inline CITATION.cff]: @danielskatz Do you want your affiliation as NCSA like it is now or should it be University of Illinois at Urbana-Champaign?

Comment (danielskatz) [inline CITATION.cff]:
```suggestion
  affiliation: "University of Illinois at Urbana-Champaign"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- family-names: "Neubauer"
+  given-names: "Mark"
+  orcid: "https://orcid.org/0000-0001-8434-9274"
+  affiliation: "University of Illinois at urbana-Champaign"
```
Comment (danielskatz) [inline CITATION.cff]:
```suggestion
  affiliation: "University of Illinois at Urbana-Champaign"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  orcid: "https://orcid.org/0000-0003-0182-7088"
+  affiliation: "CERN"
+title: "Software Citation in HEP: Current State and Recommendations for the Future"
+version: 1.0.0
+repository-code: "https://github.com/matthewfeickert-talks/talk-chep-2023/releases/tag/chep-2023"
+url: "https://indico.jlab.org/event/459/contributions/11688/"
+keywords:
+  - software citation
+  - chep 2023
+license: "BSD-3-Clause"
```
Comment (danielskatz) [inline CITATION.cff:32]: CC-BY instead?

**Comment by danielskatz** [comment]
> Seems appropriate that for a talk on software citation we have a `CITATION.cff` file for the talk as well. :)
> 
> ```
> * Add CITATION.cff for CHEP 2023 talk
>    - c.f. https://indico.jlab.org/event/459/contributions/11688/
> ```

I'm not so sure, since the talk isn't code...

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,31 @@
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
```
Comment (danielskatz) [inline CITATION.cff:2]: Is this software?

File: CITATION.cff (hunk 3 above)
Comment (matthewfeickert) [reply CITATION.cff:32]: Hm, that's not what I licesned this repository under. I'll have to think if I want the talk code to be under a different license than the talk itself. My thoughts are probably not.

File: CITATION.cff (hunk 4 above)
Comment (danielskatz) [inline CITATION.cff]: Is this software?

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+authors:
+- family-names: "Feickert"
+  given-names: "Matthew"
+  orcid: "https://orcid.org/0000-0003-4124-7862"
+  affiliation: "University of Wisconsin-Madison"
+- family-names: "Katz"
+  given-names: "Daniel"
```
Comment (danielskatz) [inline CITATION.cff]:
```suggestion
  given-names: "Daniel S."
```

**Comment by matthewfeickert** [comment]
> the talk isn't code

This talk is HTML, CSS, and JavaScript.

**Comment by msneubauer** [comment]
The ORCHID ID you have for me is correct

File: CITATION.cff (hunk 4 above)
Comment (matthewfeickert) [reply CITATION.cff]: Yes, but your point stands as it isn't general use software. Could do

```
message: "Please cite the following works when referencing this repository and talk."
```

**Pull request merged**